### ISSUE TITLE
The results of T2m seem abnormal for Unet using batch size of 1

### ISSUE THREAD

**Issue by ShileiCao** [spcl/ens10#4]
The loss and metric are extremely low when using the default batch size of 1 and default learning rate of 0.01 for Unet

![Image](https://github.com/user-attachments/assets/35e7b4e4-0e4a-4295-847c-83025f516e2e)

However, the result seems normal when switching the batch size to 8 or switching the learning rate to 0.00005 for Unet.

Is there any wrong with the code?

**Comment by ShileiCao** [comment]
I find that it is because the model may find a way to make the loss extremely low by making the prediction about std (output[:, 1]) extremely low. 
The is the output of output[:, 0] and torch.exp(output[:, 1])

![Image](https://github.com/user-attachments/assets/be0638bf-cd8e-4e20-bd98-326a9c8ad368)
